Source/CarthageKit/Xcode.swift

```diff
@@ -171,14 +171,14 @@ public func schemesInProject(project: ProjectLocator) -> SignalProducer<String,
 
 /// Finds schemes of projects or workspaces, which Carthage should build, found
 /// within the given directory.
-public func buildableSchemesInDirectory(directoryURL: NSURL, withConfiguration configuration: String, forPlatforms platforms: Set<Platform> = []) -> SignalProducer<(ProjectLocator, [String]), CarthageError> {
+public func buildableSchemesInDirectory(directoryURL: NSURL, withConfiguration configuration: String, forPlatforms platforms: Set<Platform> = [], derivedDataPath: String? = nil) -> SignalProducer<(ProjectLocator, [String]), CarthageError> {
```

Comment:
I don't believe this function should take `derivedDataPath`—that shouldn't affect the output.